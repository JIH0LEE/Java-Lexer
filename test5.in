+
-
*
/
<=
>=
=
==
!=
>
<
'!'
'$'
'@'
_int
"jiho"
0
-2
