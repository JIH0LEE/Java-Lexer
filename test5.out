<OP,+>
<OP,->
<OP,*>
<OP,/>
<COM_OP,<=>
<COM_OP,>=>
<ASSIGN>
<COM_OP,==>
<COM_OP,!=>
<COM_OP,>>
<COM_OP,<>
<CHARACTER,'!'>
<CHARACTER,'$'>
<CHARACTER,'@'>
<ID,_int>
<STRING,"jiho">
<INTEGER,0>
<INTEGER,-2>
