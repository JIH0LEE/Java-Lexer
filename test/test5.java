int name=1;
int name1=name*3/2;
int name2=(name1*2)/(name+2);
int fun1(int name,int name1,int name2){
    if(true>true>true){
    name=name2;
    }
    return name;
}