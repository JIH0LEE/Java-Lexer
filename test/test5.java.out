<1,VTYPE,int>
<1,ID,name>
<1,ASSIGN,=>
<1,NUM,1>
<1,SEMI,;>
<2,VTYPE,int>
<2,ID,name1>
<2,ASSIGN,=>
<2,ID,name>
<2,OP,*>
<2,NUM,3>
<2,OP,/>
<2,NUM,2>
<2,SEMI,;>
<3,VTYPE,int>
<3,ID,name2>
<3,ASSIGN,=>
<3,LPAREN,(>
<3,ID,name1>
<3,OP,*>
<3,NUM,2>
<3,RPAREN,)>
<3,OP,/>
<3,LPAREN,(>
<3,ID,name>
<3,OP,+>
<3,NUM,2>
<3,RPAREN,)>
<3,SEMI,;>
<4,VTYPE,int>
<4,ID,fun1>
<4,LPAREN,(>
<4,VTYPE,int>
<4,ID,name>
<4,COMMA,,>
<4,VTYPE,int>
<4,ID,name1>
<4,COMMA,,>
<4,VTYPE,int>
<4,ID,name2>
<4,RPAREN,)>
<4,LBRACE,{>
<5,IF,if>
<5,LPAREN,(>
<5,BOOLSTR,true>
<5,COMP,>>
<5,BOOLSTR,true>
<5,COMP,>>
<5,BOOLSTR,true>
<5,RPAREN,)>
<5,LBRACE,{>
<6,ID,name>
<6,ASSIGN,=>
<6,ID,name2>
<6,SEMI,;>
<7,RBRACE,}>
<8,RETURN,return>
<8,ID,name>
<8,SEMI,;>
<9,RBRACE,}>
